### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mawaheb-seraj-devtech**
#### What's this PR do?

Adds the landing page placeholder.
#### Where should the reviewer start?

index.html
#### How should this be manually tested?

simply by visiting the index page for now.
#### Any background context you want to provide?

no.
#### What are the relevant tickets?

None.

### CODE REVIEW

File: App/app.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+            },{
+                xtype: 'button',
+                text: 'Lucky Guy!',
+                cls: 'myButton'
+            }]
+        }]
+    });
+
+
```
Comment (dejan-babic) [inline App/app.js]: Unnecessary empty row regarding our coding standards. Also noticed that the indention is space based and not tab based, so i guess you are not using phpStorm as IDE

File: CSS/main-css.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+/*.flex-items-default {*/
+    /*width: 300px;*/
+    /*height: 250px;*/
+    /*flex-grow: 0;*/
+    /*flex-shrink: 0;*/
+    /*flex-basis: 0;*/
+/*}*/
+
+
+
```
Comment (dejan-babic) [inline CSS/main-css.css]: We should avoid naming classes with prefix "my", please change them

**Comment by dejan-babic** [comment]
One note, I apologize I missed this on the first review, please make the folder names lowercase (camelCase if needed)

**Comment by mawaheb-seraj-devtech** [comment]
I believe all the comments are addressed, Please review, and let me know if there are any other modifications that should be done from my side.

**Comment by dejan-babic** [comment]
Ok, one more thing...
Because we don't have a proper build system, you will need to include the necessary ext libs (js&css) as part of the repository. Please do so and we can call it done

**Pull request merged**